proxy/http/HttpTransactCache.cc

```diff
@@ -1299,7 +1299,7 @@ HttpTransactCache::match_response_to_request_conditionals(HTTPHdr *request, HTTP
       ink_time_t lm_value = response->get_last_modified();
 
       // we won't return NOT_MODIFIED if Last-modified is too recent
-      if ((lm_value == 0) || (request->get_if_modified_since() < lm_value)) {
```

Comment:
I am -1 on this change.

You are failing to see how this change would modify the behavior for everyone else that uses ATS.  If you want to submit a change that would have a configuration to turn this feature off then that would be better.